FAQ.md

```diff
@@ -18,10 +19,26 @@ This page will be updated during the course of the programme following the updat
 
 See [here](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/annex-f.htm).
 
+
+## Review
+
 **The check is failing. Where is the report?**
 
 ![grafik](https://user-images.githubusercontent.com/59165496/214881281-2f428c31-2de1-48f3-8564-2a7b27fe4174.png "In the Action tab, click on *Summary*.")
 
+**I get the following message in review**
+
+- *lftm*
+
+Looks fine to me. This means the reviewer did not spot any obvious mistakes.
+
+- *The file has a lot more content than required - this approval only covers the concepts as part of Sprint x.y. It may be that the file will fail future checks and will need amendment.*
```

Comment:
only sprint x.y or also all other sprints before it?

Reply:
Good question. In my view:

- sprint x.y has other sprints as prerequisites
- checker is continuously being expanded

therefore proposing an expansion: 

... the concepts as part of Sprint x.y *and its prerequisites* ...

Reply:
Agreed.